src/metpy/calc/thermo.py

```diff
@@ -2574,3 +2574,55 @@ def specific_humidity_from_dewpoint(pressure, dewpoint):
     """
     mixing_ratio = saturation_mixing_ratio(pressure, dewpoint)
     return specific_humidity_from_mixing_ratio(mixing_ratio)
+
+
+@exporter.export
+@preprocess_xarray
+@check_units('[pressure]', '[temperature]', '[temperature]')
+def lifted_index(pressure, temperature, parcel_profile):
+    """Calculate Lifted Index from the pressure temperature and parcel profile.
+
+    Lifted index formula derived from [Galway1956] and referenced by [Doswell-Schultz2006]:
+    LI = T500 - Tp500
+    where:
+    T500 is the measured temperature at 500 hPa.
+    Tp500 is the temperature of the lifted parcel at 500 hPa.
+
+    Calculation of the lifted index is defined as the temperature difference between the
+    observed 500 hPa temperature and the temperature of a parcel lifted from the
+    surface to 500 hPa.
+
+    Parameters
+    ----------
+    pressure : `pint.Quantity`
+        The atmospheric pressure level(s) of interest, in order from highest to
+        lowest pressure.
+    temperature : `pint.Quantity`
+        The atmospheric temperature corresponding to pressure.
+    parcel_profile : `pint.Quantity`
+        The temperature profile of the parcel.
+
+    Notes:
+    ------
+    References to the lifted index equation came from:
+    Galloway 1956 Bulletin American Meteorological Society
```

Comment:
Instead of denoting the papers here, can you put in `[docs/references.rst](https://raw.githubusercontent.com/Unidata/MetPy/master/docs/references.rst)`. There should be other examples of these journals being cited in there.

If you move those, you likely won't need anything in "Notes" I think.

Reply:
Ok. Sounds good to me. You just answered my previous questions.

Reply:
Reference links in the notes for thermo.py was moved to the references.rst document. Removed notes section in thermo.py file. I am not familiar with the notation for references in the document. Used other examples for creating new references.